src/o365/spo/commands/contenttype/contenttype-field-remove.ts

```diff
@@ -0,0 +1,307 @@
+import auth from '../../SpoAuth';
+// import { ContextInfo, ClientSvcResponse, ClientSvcResponseContents } from '../../spo';
+// import config from '../../../../config';
+import * as request from 'request-promise-native';
+import commands from '../../commands';
+import GlobalOptions from '../../../../GlobalOptions';
+import {
+  CommandOption, // CommandValidate, CommandTypes,
+  CommandTypes,
+  CommandValidate,
+  CommandError
+} from '../../../../Command';
+import SpoCommand from '../../SpoCommand';
+import Utils from '../../../../Utils';
+import { Auth } from '../../../../Auth';
+import { ContextInfo, ClientSvcResponse, ClientSvcResponseContents } from '../../spo';
+import config from '../../../../config';
+// import { FieldLink } from './FieldLink';
+// REST URL
+// POST http://<sitecollection>/<site>/_api/web/lists(listid)/contenttypes(contenttypeidcontenttypeid})/fieldlinks(fieldlinkid)/deleteObject()
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+
+interface Options extends GlobalOptions {
+  contentTypeId: string;
+  fieldLinkId: string;
+  webUrl: string;
+  listTitle?: string;
+  updateChildContentTypes?: boolean;
+}
+
+class SpoContentTypeFieldRemoveCommand extends SpoCommand {
+  private requestDigest: string = '';
+  private webId: string = '';
+  private siteId: string = '';
+  private siteAccessToken: string = '';
+  private listId: string = '';
+
+  public get name(): string {
+    return `${commands.CONTENTTYPE_FIELD_REMOVE}`;
+  }
+
+  public get description(): string {
+    return 'Remove a site column reference from a site or list content type';
+  }
+
+  public types(): CommandTypes | undefined {
+    return {
+      string: ['i', 'contentTypeId', 'f', 'fieldLinkId']
+    };
+  }
+
+  public commandAction(cmd: CommandInstance, args: CommandArgs, cb: (err?: any) => void): void {
+    const resource: string = Auth.getResourceFromUrl(args.options.webUrl);
```

Comment:
Missing the prompt for confirmation before actually removing the field. See other remove commands for how this is implemented.